src/main/java/seedu/address/logic/commands/DeleteEventCommand.java

```diff
@@ -90,6 +90,15 @@ public CommandResult execute(Model model) throws CommandException {
 
         logger.log(Level.INFO, "Target Event deleted from event set. New Event set: " + newEventSet);
 
+        editPatientDescriptor.setName(patientToDeleteEvent.getName());
+        editPatientDescriptor.setTags(patientToDeleteEvent.getTags());
+        editPatientDescriptor.setPatientHospitalId(patientToDeleteEvent.getPatientHospitalId());
+        editPatientDescriptor.setPreferredName(patientToDeleteEvent.getPreferredName());
+        editPatientDescriptor.setFoodPreferences(patientToDeleteEvent.getFoodPreferences());
+        editPatientDescriptor.setFamilyConditions(patientToDeleteEvent.getFamilyConditions());
+        editPatientDescriptor.setHobbies(patientToDeleteEvent.getHobbies());
+        editPatientDescriptor.setName(patientToDeleteEvent.getName());
+        editPatientDescriptor.setTags(patientToDeleteEvent.getTags());
```

Comment:
I think you need to look at the recent PR #165 and resolve the conflict properly as I removed this part of the code!

Reply:
Good catch! Sorry for the conflicts changes made!